pkgs/development/tools/oh-my-posh/default.nix

```diff
@@ -1,25 +1,39 @@
-{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, installShellFiles
+}:
 
 buildGoModule rec {
   pname = "oh-my-posh";
-  version = "12.22.0";
+  version = "12.25.0";
 
   src = fetchFromGitHub {
     owner = "jandedobbeleer";
     repo = pname;
-    rev = "v${version}";
-    hash = "sha256-UIoMJjZrU9aDGnDx5K2SJO6lBgivMqGUDbtTfF3WfNE=";
+    rev = "refs/tags/v${version}";
```

Comment:
why are we hardcoding `refs/tags/`?

Reply:
See https://discourse.nixos.org/t/why-is-the-r-ryantm-bot-using-refs-tags-in-the-version/20519

Reply:
I am not fundamentally opposed to this change, but it seems to only be done for >10% of `fetchFromGitHub` uses, and applied inconsistently, as it was not called out during the initial review of this package. can we get something added to the official documentation, or at least a change drafted to ensure a sense of consensus for this new style?